python/sglang/lang/ir.py

```diff
@@ -150,8 +150,8 @@ def run(
         self,
         *args,
         max_new_tokens: int = 128,
-        stop: Union[str, List[str]] = [],
-        stop_token_ids: Optional[List[int]] = [],
+        stop: Union[str, List[str]] = None,
```

Comment:
Also consider modifying the default arguments in `run_batch`?

Reply:
sorry I realized I overwrite your commit. I can replace all `= []` to `= None` in another PR. we can keep this PR just for the stop token.

Reply:
Never mind, it seems all the mutable default arguments are related to `stop` or `stop_ids`...